src/main/java/org/opensrp/repository/postgres/mapper/custom/xml/CustomLocationMetadataMapper.xml

```diff
@@ -410,4 +410,43 @@
 
 	</select>
 
+    <select id="selectLocationTreeWithAncestors"
```

Comment:
I think is similar to `selectLocationHierachy` could we use that method

Reply:
Yes, tested with `selectLocationHierachy` method. It would work.